<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Common</class>
 <widget class="QMainWindow" name="Common">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>常规设置</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>关闭程序</string>
      </property>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>80</y>
         <width>120</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>退出时二次确认</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_close">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>130</y>
         <width>180</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 170, 255);</string>
       </property>
       <property name="text">
        <string>退出程序</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_min">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>80</y>
         <width>180</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 127);</string>
       </property>
       <property name="text">
        <string>最小化到托盘，不退出程序</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>Common</receiver>
   <slot>on_checked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_checked()</slot>
 </slots>
</ui>
